### ISSUE TITLE
Restoring rules state from savepoint or checkpoint

### ISSUE THREAD

**Issue by pranjal0811** [haoch/flink-siddhi#38]
Hi Hoach, 

I am using flink siddhi wrapper for the dynamic rule. when I try to restore my job from savepoint or restarting the job from failure by checkpointing. I am not able to recover the rules from the state. 

Do this wrapper support state management? So, that we can recover the rule from the savepoint. 

Thanks,
Pranjal

**Comment by haoch** [comment]
Yes, it supports state management. What's the error by your side? The recovering error is caused from flink side or siddhi internal?
